### ISSUE TITLE
Filter Column unable to paste copied value

### ISSUE THREAD

**Issue by Dixuss** [microsoft/vscode-data-wrangler#136]
Type: <b>Bug</b>

# Behaviour

## Expected vs. Actual

XXX

## Steps to reproduce:

1. XXX

# Diagnostic data

-   Jupyter extension version: 2024.1.1
-   Python extension version: 2024.0.1
-   .NET Install Tool for Extension Authors extension version: 2.0.1
-   Python package dependencies:

```json
{
  "installed": {
    "pandas": "2.1.4"
  },
  "required": {
    "pandas": "0.25.2"
  },
  "unsatisfied": []
}
```

Extension version: 0.26.0
VS Code version: Code 1.86.2 (Universal) (903b1e9d8990623e3d7da1df3d33db3e42d80eda, 2024-02-13T19:42:13.651Z)
OS version: Darwin x64 23.2.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700B CPU @ 3.20GHz (12 x 3200)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|8.00GB (0.02GB free)|
|Process Argv|--crash-reporter-id 2edde177-7d80-476f-81cc-ee59e174faa0|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vscod805:30301674
binariesv615:30325510
vsaa593:30376534
py29gd2263:30899288
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
h48ei257:30962250
pythontbext0:30879054
accentitlementsc:30887149
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
pydisww1:30959800
b1063948:30958189
3ef8e399:30964150
bg6jg535:30958357

```

</details>

**Comment by pwang347** [comment]
Hi @Dixuss, thanks for reporting this issue! I can confirm that this repros (on a Mac device) as well. Will investigate this and let you know what I find.

**Comment by pwang347** [comment]
Hi @Dixuss, this issue is now resolved in the latest pre-release version 1.1.1. Please give it a try and if the issue persists let me know. Thanks!
